build.sbt

```diff
@@ -86,7 +86,9 @@ lazy val deriving = crossProject(JSPlatform, JVMPlatform)
   .settings(mimaSettings)
   .settings(
      mimaBinaryIssueFilters ++= Seq(
-       ProblemFilters.exclude[ReversedMissingMethodProblem]("shapeless3.deriving.internals.ErasedProductInstances.erasedProject")
+       ProblemFilters.exclude[ReversedMissingMethodProblem]("shapeless3.deriving.internals.ErasedInstances.erasedMapK"),
+       ProblemFilters.exclude[ReversedMissingMethodProblem]("shapeless3.deriving.internals.ErasedProductInstances.erasedProject"),
+       ProblemFilters.exclude[ReversedMissingMethodProblem]("shapeless3.deriving.internals.ErasedProductInstances.erasedMapK")
```

Comment:
Don't we need an exclusion for `shapeless3.deriving.internals.ErasedCoproductInstances.erasedMapK` as well?

Reply:
No, because it's final. I guess if we made `Instances` and `ProductInstances` sealed we would need no exclusions at all.

Reply:
I'm fine with making it sealed, but shouldn't [this](https://github.com/lightbend/mima/pull/583) mean none of these should be necessary?

Reply:
Not for Scala 3: https://github.com/lightbend/mima/pull/583#issuecomment-734965304

Reply:
Ahh ... right. Then this is good to go :smile: